source/patterns/@aws-solutions-constructs/aws-openapigateway-lambda/lib/index.ts

```diff
@@ -164,6 +164,9 @@ export class OpenApiGatewayToLambda extends Construct {
     let lambdaCounter = 0;
 
     this.apiLambdaFunctions = props.apiIntegrations.map(apiIntegration => {
+      if (apiIntegration.existingLambdaObj && apiIntegration.lambdaFunctionProps) {
```

Comment:
I like where you placed the error checking, but this is also done in the else statement below - see [here](https://github.com/awslabs/aws-solutions-constructs/blob/main/source/patterns/%40aws-solutions-constructs/aws-openapigateway-lambda/lib/index.ts#L176)